.github/workflows/build-graphscope-wheels-linux.yml

```diff
@@ -193,10 +196,13 @@ jobs:
     - name: Prepare Environment
       shell: bash
       run: |
-        yum install -y gcc gcc-c++ make cmake
+        dnf install -y gcc gcc-c++ openssl-devel bzip2-devel libffi-devel wget make cmake git
 
-        # install git and python3
-        yum install -y git python3-pip python3-devel
+        # install python3.7
+        cd /tmp && wget https://www.python.org/ftp/python/3.7.9/Python-3.7.9.tgz
+        tar xzf Python-3.7.9.tgz
+        cd Python-3.7.9 && ./configure --enable-optimizations --enable-shared
+        make install
```

Comment:
Could we install python 3.8 (I think it should be possible, as https://forums.centos.org/viewtopic.php?p=319600) using dnf or yum instead?

Building a custimized python for every CI runs seems not a good idea.

Reply:
done~